docs/examples/networking/vcn_default/vcn.tf

```diff
@@ -0,0 +1,128 @@
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "compartment_ocid" {}
+variable "region" {}
+
+provider "oci" {
+  tenancy_ocid = "${var.tenancy_ocid}"
+  user_ocid = "${var.user_ocid}"
+  fingerprint = "${var.fingerprint}"
+  private_key_path = "${var.private_key_path}"
+  region = "${var.region}"
+}
+
+resource "oci_core_virtual_network" "vcn1" {
+  cidr_block = "10.0.0.0/16"
+  dns_label = "vcn1"
+  compartment_id = "${var.compartment_ocid}"
+  display_name = "vcn1"
```

Comment:
nitpick: I like the naming convention of prepending the name with TF. It easily lets find the resources created from TF example